lib/unittest.bzl

```diff
@@ -590,3 +662,9 @@ analysistest = struct(
     target_bin_dir_path = _target_bin_dir_path,
     target_under_test = _target_under_test,
 )
+
+loadingtest = struct(
+    make = _loading_make,
+    asserts = _loading_asserts,
```

Comment:
Nit: please rename to "assert", not "asserts" for consistency with the `asserts` struct defined above (in other words, we want `assert` to be a method and `asserts` to be a struct/namespace).

Reply:
I can't do this.  assert is a keyword in starlark and fails. For now I've renamed to 'equals' to mirror the function in 'asserts'

I'm open to other suggestions.